src/js/controller/instream-adapter.js

```diff
@@ -199,9 +199,9 @@ define([
                 playlist = [item];
             }
 
-            var _this = this;
+            _this = this;
```

Comment:
You can delete the line. `loadItem` is only called from VAST in `_playInstream` and here in `_loadNextItem`.